package.json

```diff
@@ -40,18 +40,18 @@
   "devDependencies": {
     "@semantic-release/commit-analyzer": "^8.0.1",
     "@semantic-release/git": "^9.0.0",
-    "@semantic-release/github": "^7.1.1",
+    "@semantic-release/github": "^7.2.0",
```

Comment:
Maybe you should rebase your PR. I don't think these changes come from your PR.

Reply:
@hi-rustin Maybe I should relearn Git. 😭